build.gradle

```diff
@@ -1,58 +1,24 @@
 buildscript {
   repositories {
-    mavenCentral()
+      jcenter()
+      google()
   }
-
   dependencies {
-    classpath 'com.android.tools.build:gradle:1.3.1'
+    classpath 'com.android.tools.build:gradle:3.2.1'
```

Comment:
Why did you choose this version instead of the most recent stable version (3.3.1)?

Reply:
This need basic gradle toto updated to 4.10.1.from current version is 4.6 . The deep link SDK is using 4.6 so thought of following the same